### ISSUE TITLE
Part tags!

### ISSUE THREAD

**Issue by aw1621107** [KSP-RO/RealismOverhaul#1206]
List of things that should be automatically tagged by RO.

- RealFuels
  - [x] Fuel/oxidizer used by engines
  - [ ] Type of fuel tank
- [ ] Manufacturer?
- [ ] Real-life rockets part was used by?
- Others?

**Comment by aw1621107** [comment]
I *think* MM doesn't support reading/processing values (e.g. comparing two values for equality), so I don't think manufacturer/restarts/throttleable can be tagged in this way

**Comment by aw1621107** [comment]
Never mind, @NathanKell is smarter than I am.

**Comment by jwvanderbeck** [comment]
You can always do things by hand too if necessary you know :)

**Comment by aw1621107** [comment]
@jwvanderbeck True, but I'd rather go for the low-hanging fruit first. Don't want to sift through a bajillion engines at this time adding more specific info

**Comment by Capkirk123** [comment]
This seems like a nice idea, and should be fairly automatable by MM.
